internal/provider/provider.go

```diff
@@ -45,6 +45,7 @@ func Provider() terraform.ResourceProvider {
 			"unifi_user_group":     resourceUserGroup(),
 			"unifi_user":           resourceUser(),
 			"unifi_wlan":           resourceWLAN(),
+			"unifi_port_forward":   resourcePortForward(),
```

Comment:
Just putting these in alpha order
```suggestion
			"unifi_port_forward":   resourcePortForward(),
			"unifi_user_group":     resourceUserGroup(),
			"unifi_user":           resourceUser(),
			"unifi_wlan":           resourceWLAN(),
```